help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
 
 # Getting Started with Building on Datagrok
 
-Welcome to [Datagrok](https://www.datagrok.ai), the next generation data platform!
+Welcome to Datagrok, the next-generation data platform!
 
-This is a guide to help you jump-start using Datagrok as an engineer, extending Datagrok
-with plugins and building applications on top of it. During this Getting Started,
-post your questions and suggestions at our [Community Forum](https://community.datagrok.ai/).
-We usually respond within less than a day.
+This guide will help you jump-start using Datagrok as an engineer. In short, what you can do with Datagrok is extend it
+with plugins or build your applications on top of Datagrok.
+
+If you have any questions or suggestions after reading this Getting Started guide, go to our [Community Forum] &mdash; 
+typically respond within 24 hours.
 
 ## Let's start
 
-*Take it fast.* If you are already somewhat knowledgeable with the platform, maybe due to
```

Comment:
@AndSviat why do we prefer to remove the two directions here?
The idea was as follows: we saw that some developers preferred to skip directly to the exercises. If that's the case, at least we are making sure they watch some key videos. Otherwise we direct them to the regular track.
Do you suggest to remove this story line, or maybe it's possible to simplify it?